test_isort.py

```diff
@@ -1767,6 +1767,18 @@ def test_import_by_paren_issue_375():
     assert SortImports(file_contents=test_input).output == 'from .models import Bar, Foo\n'
 
 
+def test_import_by_paren_issue_460():
+    """Test to ensure isort can doesnt move comments around """
+    test_input = """
```

Comment:
Quick comment: using implicit string literal continuations using parentheses seem to be preferred in the rest of the tests.